alchemy/src/cloudflare/worker.ts

```diff
@@ -375,6 +375,14 @@ export interface BaseWorkerProps<
      */
     cpu_ms?: number;
   };
+
+  /**
+   * Tail consumers that will receive execution logs from this worker
+   */
+  tailConsumers?: Array<{
+    /** Name of the tail worker service */
+    service: string;
+  }>;
```

Comment:
What services are supported? Are they meant to be other workers? If so, then it should be `(Worker | {service: string})[]`

Reply:
Yes, that is the polishing we are working on right now lol